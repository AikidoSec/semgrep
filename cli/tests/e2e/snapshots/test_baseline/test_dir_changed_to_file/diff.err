               
               
┌─────────────┐
│ Scan Status │
└─────────────┘
  Scanning 1 file tracked by git with 1 Code rule, 0 Supply Chain rules, 0 Pro  
  rules:                                                                        
            
  CODE RULES
                                                                                
  Language   Rules   Files          Origin   Rules                              
 ──────────────────────────        ────────────────                             
  python         1       1          Custom       1                              
                                                                                
                    
  SUPPLY CHAIN RULES
                                                                         
  💎 Sign in with `semgrep login` and run `semgrep ci` to find dependency
     vulnerabilities and advanced cross-file findings.                   
                                                                         
          
  PROGRESS
   
  Current version has 1 finding.

Creating git worktree from 'HEAD^' to scan baseline.
  Will report findings introduced by these commits (may be incomplete for shallow checkouts):
    * 321d62c commit #2

               
               
┌─────────────┐
│ Scan Status │
└─────────────┘
  Scanning 1 file tracked by git with 1 Code rule, 0 Supply Chain rules, 0 Pro  
  rules:                                                                        
            
  CODE RULES
                                                                                
  Language   Rules   Files          Origin   Rules                              
 ──────────────────────────        ────────────────                             
  python         1       1          Custom       1                              
                                                                                
                    
  SUPPLY CHAIN RULES
                                                                         
  💎 Sign in with `semgrep login` and run `semgrep ci` to find dependency
     vulnerabilities and advanced cross-file findings.                   
                                                                         
          
  PROGRESS
   
                
                
┌──────────────┐
│ Scan Summary │
└──────────────┘
Some files were skipped or only partially analyzed.
  Scan was limited to files changed since baseline commit.

Ran 1 rule on 1 file: 0 findings.
