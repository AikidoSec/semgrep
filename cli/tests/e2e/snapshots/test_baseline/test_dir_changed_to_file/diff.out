x = 23478921
x = 23478921
x = 23478921
x = 23478921
x = 23478921
x = 23478921
x = 23478921
x = 23478921
